release-notes.md

```diff
@@ -8,6 +8,20 @@ description: Release notes and change logs for the Adobe Experience Platform Mob
 
 Please take a moment to fill out a [short survey](https://www.surveymonkey.com/r/AEPDocs) on how we can better assist you with enabling Adobe Experience Cloud solutions and services on your mobile apps.
 
+##  March 12, 2020
+
+The following updates were made in this release:
+
+### iOS Core 2.5.1
```

Comment:
You don't need to list the Identity changes under Core as well. Only the changes specific to Core should get listed under Core. It is sufficient to just list the Identity changes under Identity.